### PR TITLE
Change the way span of label is added to the label widget

### LINKED ISSUE #525: Arbitrary JS execution while rendering label from list

### PR DESCRIPTION
**Pull request by vidya-ram**
Fixes #525

### CODE REVIEW

File: funnel/assets/js/proposal.js [hunk 1]
```diff
@@ -228,12 +228,16 @@ export const LabelsWidget = {
   updateLabels(label = '', attr = '', action = true) {
     if (action) {
       if (label !== attr) {
-        $(`.label[data-labeltxt="${attr}"]`).remove();
+        $(`.label[data-labeltxt='${attr}']`).remove();
       }
-      const span = `<span class="label mui--text-caption mui--text-bold" data-labeltxt="${attr}">${label}</span>`;
-      $('#label-select').append(span);
+      let labelSpan = $(
+        '<span class="label mui--text-caption mui--text-bold"></span'
```
Comment (jace) [inline funnel/assets/js/proposal.js]: Missing `>`

File: funnel/assets/js/proposal.js [hunk 2]
```diff
@@ -228,12 +228,16 @@ export const LabelsWidget = {
   updateLabels(label = '', attr = '', action = true) {
     if (action) {
       if (label !== attr) {
-        $(`.label[data-labeltxt="${attr}"]`).remove();
+        $(`.label[data-labeltxt='${attr}']`).remove();
```
Comment (jace) [inline funnel/assets/js/proposal.js]: Any reason for the change in quotes?

File: funnel/assets/js/proposal.js (hunk 1 above)
Comment (vidya-ram) [reply funnel/assets/js/proposal.js]: Fixed

File: funnel/assets/js/proposal.js (hunk 2 above)
Comment (vidya-ram) [reply funnel/assets/js/proposal.js]: Default is single quotes

**Pull request merged**